### ISSUE TITLE
R and parallel computing

### ISSUE THREAD

**Issue by n0542344** [eddelbuettel/t4#5]
tl;dr: An episode on using parallelization in R (local system multithreading, using R on GPUs and working with HPCs) would be awesome!

- - -  

I'm not sure if there is an upper limit on the amount of issues I'm allowed to post ;)

Working with multithreading is a huge timesaver. On GNU/Linux (and maybe other UNIXoid OS) it's not that tricky to get started with that: 

    doParallel::registerDoParallel(cores = (parallel::detectCores() - 2))

already get's you quite far. But performing computations on local GPUs or even working on/with HPCs and other distributed systems did not seem that trivial to me. 

I'm unsure if that fits into the lists of topics you planned to do, but I'd definitely love that.

Best wishes, Alex

**Comment by eddelbuettel** [comment]
Thanks for the sugggestion. That is pretty far out in advanced stuff and maybe too high a hurdle for T^4 and it's "five+ mins if we can" length.  I have to think about it.

No upper limit on suggestions.  That's why we have this repo :)
